custom_model_runner/datarobot_drum/resource/deployment_config_helpers.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+from datarobot_drum.drum.exceptions import DrumCommonException
+from datarobot_drum.drum.common import TargetType
+
+
+def parse_validate_deployment_config_file(filename):
+    if filename is None:
```

Comment:
Maybe better `if not filename`? (to include an empty string)

Reply:
filename existence is checked in the arg parse validator